synapse/lib/layer.py

```diff
@@ -177,6 +180,74 @@ class Layer(s_cell.Cell):
             # yield buid, form, prop, valu
             yield (buid, )
 
+    # TODO: Hack until we get interval trees pushed all the way through
+    def _cmprIval(self, item, othr):
```

Comment:
Is there a better place to put this? The comparison needs to be done here so we don't shunt 1000 needs onto the wire up to the snap, only to be reduced down to a handful, but it's an ugly hack to have this here.

Reply:
Certainly _cmprIval is in the right place as it will be overridden by layer subclasses. The _liftBy* methods are more debatable, but still, it should be in the same place as the regex lifts which are here.